examples/Rules-method-maxDose-IncrementsAbsolute.R

```diff
@@ -0,0 +1,17 @@
+
+# Create the data
+data <- Data(x=c(0.1, 0.5, 1.5, 3, 6, 8, 8, 8),
```

Comment:
please style the code with styler, and check it with linter.

Reply:
styled the new code with `stylr`.

`lintr` reported an error regarding a missing `nolint start`.  I added one at line 983.

`lintr` then reported  several "variable and function name style should be snake_case or symbols" warnings for the new code.  This is consistent with the same warnings reported elsewhere.  I assume this is acceptable.